src/shared/components/oncoprint/DeltaUtils.ts

```diff
@@ -1096,6 +1101,16 @@ function transitionGeneticTrack(
         return;
     } else if (nextSpec && !prevSpec) {
         // Add track
+        const createCustomOncoprintTooltip = getServerConfig().customJsFunctions
```

Comment:
using a ternary here is very confusing.  just use if/else.   next developer will thank you for it.

Reply:
Fixed in 
96026be